server/integration_tests/test_data/sdg/query_1/chart_config.json

```diff
@@ -11,14 +11,14 @@
                     "statVarKey": [
                       "sdg/SI_POV_DAY1"
                     ],
-                    "title": "Proportion of Population Below International Poverty Line in Countries of Africa (${date})",
+                    "title": "Population Below International Poverty Line in Countries of Africa (${date})",
```

Comment:
For one of these with SDG response, can we add another mode that sets `dc=sdg` as validation that the prior titles are unaffected?

Reply:
the sdg tests in explore_test.py actually already set dc=sdg, but I've added an sdg test that doesn't set dc there!